src/test/java/cat/udl/eps/softarch/demo/steps/StepRegisterAdoption.java

```diff
@@ -0,0 +1,34 @@
+package cat.udl.eps.softarch.demo.steps;
+
+import cat.udl.eps.softarch.demo.domain.Pet;
+import cat.udl.eps.softarch.demo.domain.User;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class StepRegisterAdoption {
+    private Pet pet;
+
+    private User user;
+
+    @Given("The user is logged in with username \"username\" and password \"password\"")
```

Comment:
This method is not necessary because it is already implemented in RegisterStepDefs.java.